Dockerfile

```diff
@@ -8,3 +8,8 @@ WORKDIR /go/src/github.com/sendgrid/sendgrid-go
 COPY . .
 
 RUN make install
+
+# Use the last version of testify that works for older go versions, and then
+# re-install to update dependencies.
+RUN (cd /go/src/github.com/stretchr/testify && git checkout v1.6.0)
+RUN make install
```

Comment:
why run make install again?

Reply:
> to update dependencies

Wasn't working without it. Assuming older version of testify had different dependencies that needed to be pulled in. This was the simplest way to do it.